Makefile

```diff
@@ -18,16 +18,20 @@ DOWNLOADFILES := $(BIN_DOWNLOAD_FILEPATH) $(LOG_DOWNLOAD_FILEPATH)
 
 help:
 	@echo 'Cleaning targets:'
-	@echo '  clean                      - clean the cache artifacts'
+	@echo '  clean                      - clean the cache artifacts and generated files'
 	@echo 'Generic targets:'
 	@echo '* all                        - build solidity code. To build from a clean clone, run: make submodules all'
 	@echo '  build                      - build solidity code'
-	@echo '  dep                        - install npm packages'
-	@echo '  depclean                   - remove npm packages'
-	@echo '  deploy                     - deploy to local node'
-	@echo '  generate                   - generate solidity code from cpp and template'
-	@echo '  test                       - test both binary files and log files'
-	@echo '  coverage                   - generate coverage report for html view'
+	@echo '  generate-all               - generate all solidity code'
+	@echo '  generate-step              - generate solidity-step code from cpp'
```

Comment:
I am confused which task will regenerate `ready_src`, I could only regenerate it with `make all`, while doing `make generate-all` after `make all` revert `ready_src` back.

And if I do not regenerated it I got the following test errors:

```
Running 7 tests for test/AccessLogs.t.sol:AccessLogsTest
[PASS] testEndianSwap() (gas: 1797)
[PASS] testReadWord() (gas: 169743)
[FAIL. Reason: Call did not revert as expected] testReadWordRoot() (gas: 172653)
[FAIL. Reason: Call did not revert as expected] testReadWordValue() (gas: 172515)
[PASS] testWriteWord() (gas: 169240)
[FAIL. Reason: Call did not revert as expected] testWriteWordRootPosition() (gas: 172278)
[FAIL. Reason: Call did not revert as expected] testWriteWordRootValue() (gas: 175815)
Test result: FAILED. 3 passed; 4 failed; 0 skipped; finished in 4.31ms

Running 1 test for test/Memory.t.sol:MemoryTest
[PASS] testStrideAlignment() (gas: 497636309)
Test result: ok. 1 passed; 0 failed; 0 skipped; finished in 62.26ms
Ran 3 test suites: 6 tests passed, 4 failed, 0 skipped (10 total tests)

Failing tests:
Encountered 4 failing tests in test/AccessLogs.t.sol:AccessLogsTest
[FAIL. Reason: Call did not revert as expected] testReadWordRoot() (gas: 172653)
[FAIL. Reason: Call did not revert as expected] testReadWordValue() (gas: 172515)
[FAIL. Reason: Call did not revert as expected] testWriteWordRootPosition() (gas: 172278)
[FAIL. Reason: Call did not revert as expected] testWriteWordRootValue() (gas: 175815)
```